Update procedure and add example request (#12404)
cloudflare/cloudflare-docs: content/cache/how-to/cache-rules/create-api.md

    * kind: `zone`
    * phase: `http_request_cache_settings`
3. Use the [Update a zone ruleset](/api/operations/updateZoneRuleset) operation to add a cache rule to the list of ruleset rules. Alternatively, include the rule in the [Create a zone ruleset](/api/operations/createZoneRuleset) request mentioned in the previous step.

> 4. Use [Update a zone ruleset rule](/api/operations/updateZoneRulesetRule) operation to update an existing rule within the ruleset. For an example, refer to the section below.

Reviewer: ```suggestion
4. (Optional) To update an existing cache rule, use the [Update a zone ruleset rule](/api/operations/updateZoneRulesetRule) operation. For an example, refer to the section below.
```
I'd make this step optional (or not part of the numbered list), since the procedure is about creating a cache rule.

{{</details>}}

{{<details header="Example: Update an existing rule">}}

> 

Reviewer: We should probably say here what we will be editing: the rule position and the rule's `ref` attribute (since other fields look the same as in the example above). We could change the description instead of the ref, though.

```bash
---
header: Request
---
curl --request PATCH \

> https://api.cloudflare.com/client/v4/zones/{zone_id}/rulesets/{ruleset_id} \

Reviewer: ```suggestion
https://api.cloudflare.com/client/v4/zones/{zone_id}/rulesets/{ruleset_id}/rules/{rule_id} \
```
The rule ID is provided in the URL and not in the body.

--header "Authorization: Bearer <API_TOKEN>" \
--header "Content-Type: application/json" \
--data '{
      "expression": "(http.host eq \"example.com\")",
      "description": "cache everything for example.com",
      "action": "set_cache_settings",
      "action_parameters": {
        "cache": true
      }
      "enabled": true,

>       "id": "3a03d665bac047339bb530ecb439a90d",

Reviewer: ```suggestion
```
